packages/react-jss/src/compose.js

```diff
@@ -1,3 +1,5 @@
+// @flow
+type Classes = {[string]: string}
```

Comment:
Should be also imported from `jss` https://github.com/cssinjs/jss/blob/master/packages/jss/src/index.js#L33

Reply:
Fixed